src/Nest/CommonAbstractions/ConnectionSettings/ConnectionSettingsBase.cs

```diff
@@ -273,9 +278,9 @@ public TConnectionSettings InferMappingFor<TDocument>(Func<ClrTypeMappingDescrip
 		/// </summary>
 		/// <param name="documentType">The type of the POCO you wish to configure</param>
 		/// <param name="selector">describe the POCO configuration</param>
-		public TConnectionSettings InferMappingFor(Type documentType, Func<ClrTypeMappingDescriptor, IClrTypeMapping> selector)
+		public TConnectionSettings DefaultsFor(Type documentType, Func<ClrTypeDefaultsDescriptor, IClrTypeDefaults> selector)
```

Comment:
Not sure on the name, it doesn't quite feel descriptive enough. Maybe `DefaultMappingFor`?